src/Symfony/Bundle/TwigBundle/Extension/ActionsExtension.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @deprecated Deprecated in 2.2, to be removed in 3.0.
+ * @deprecated since version 2.2, to be removed in 3.0.
```

Comment:
Looks like it is as per https://github.com/fabpot/symfony/commit/b957f44110fe4193eac94506b7fdc67a2ba62ed9. The strange thing is the introduced deprecation notice in the construct method saying passing a `ContainerInterface` instance is deprecated since version 2.7 and will be removed in 3.0. The whole class is already deprecated and will be removed in 3.0, which is confusing.